uint8 fault
bool reset
bool standby
bool object
bool init
bool activating
uint8 status
bool at_requested_pos
bool in_motion
bool open_contact
bool closed_contact
uint8 current_pos
uint8 requested_pos
uint8 current
uint8 force
uint8 speed

